src/jit/emitxarch.cpp

```diff
@@ -321,35 +322,39 @@ bool IsExtendedReg(regNumber reg, emitAttr attr)
 #endif
 }
 
-// Amd64: Since XMM registers overlap with YMM registers, this routine
+// Since XMM registers overlap with YMM registers, this routine
 // can also used to know whether a YMM register in case of AVX instructions.
```

Comment:
Can you fix this comment so that it makes sense:
"can also used to know whether"